### DISCUSSION TITLE
Clicking somewhere inside a ScrollViewer focuses first control contained in the ScrollViewer

### DISCUSSION THREAD

**Discussion by oliverw** [microsoft/microsoft-ui-xaml#8227] [Q&A]
Clicking somewhere inside a ScrollViewer focuses first control contained in the ScrollViewer. Is there a where to prevent this extremely irritating behavior?

**Comment by mikernet** [comment]
Did you ever figure it out?

**Comment by eduardobragaxz** [comment]
Have you tried in 1.6? It's listed as having been fixed.
